types/react-leaflet/react-leaflet-tests.tsx

```diff
@@ -83,89 +101,100 @@ export class AnimateExample extends Component<any, any> {
 }
 
 // bounds.js
-const outer: Array<[number, number]> = [
+const outer: Leaflet.LatLngBoundsLiteral = [
     [50.505, -29.09],
     [52.505, 29.09],
 ];
-const inner: Array<[number, number]> = [
+const inner: Leaflet.LatLngBoundsLiteral = [
     [49.505, -2.09],
     [53.505, 2.09],
 ];
 
-export class BoundsExample extends Component<any, any> {
+interface BoundsExampleState {
+    bounds: Leaflet.LatLngBoundsLiteral;
+}
+
+export class BoundsExample extends Component<undefined, BoundsExampleState> {
     state = {
         bounds: outer,
     };
 
     onClickInner = () => {
         this.setState({ bounds: inner });
-    }
+    };
 
     onClickOuter = () => {
         this.setState({ bounds: outer });
-    }
+    };
 
     render() {
         return (
             <Map bounds={this.state.bounds}>
                 <TileLayer
                     attribution='&copy <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
                     url='http://{s}.tile.osm.org/{z}/{x}/{y}.png'
-                    />
+                />
                 <Rectangle
                     bounds={outer}
                     color={this.state.bounds === outer ? 'red' : 'white'}
-                    onclick={this.onClickOuter}
-                    />
+                    onClick={this.onClickOuter}
+                />
                 <Rectangle
                     bounds={inner}
                     color={this.state.bounds === inner ? 'red' : 'white'}
-                    onclick={this.onClickInner}
-                    />
+                    onClick={this.onClickInner}
```

Comment:
I was experimenting; probably shouldn't have committed that change. I've since changed it back to the lowercase version.